pkg/ui/workspaces/cluster-ui/src/insights/indexActionBtn.spec.ts

```diff
@@ -69,6 +69,13 @@ describe("Create index name", () => {
       expected:
         "CREATE INDEX IF NOT EXISTS t_expr_storing_rec_idx ON t ((i + l)), (j + k), a) STORING (k)",
     },
+    {
+      name: "handles table names containing quotes, doesn't include quotes in idx name",
+      query:
+        'CREATE INDEX ON defaultdb.public."offers"."startdate" (n) STORING (b);',
```

Comment:
Can you also add test cases for if the table name or column names are quoted due to spaces -
e.g. 
```
CREATE INDEX ON "my table" ("my col");
```

Reply:
and it also would be worth testing tables that have quotes in the name:
```
create table "my""table" ("with""quote" int);
```

Reply:
Tests added for each, thank you!